lib/ssl/src/ssl_handshake.erl

```diff
@@ -1706,23 +1705,44 @@ select_hashsign(#certificate_request{certificate_types = Types}, Cert, _, Versio
             ?ALERT_REC(?FATAL, ?INSUFFICIENT_SECURITY, no_suitable_signature_algorithm)
     end.
 
-
 do_select_hashsign(HashSigns, PublicKeyAlgo, SupportedHashSigns) ->
-    case lists:filter(fun({H, rsa_pss_pss = S} = Algos) when S == PublicKeyAlgo ->
+    case lists:filter(fun({H, rsa_pss_pss = S} = Algos) when S == PublicKeyAlgo -> %% Backported from TLS-1.3, but only TLS-1.2 configured
```

Comment:
It might be correct and present before this PR, but I find it hard to read when such complex lists:filter expression is placed between `case` and `of`.
One way for improving the readability could be to store filter result in variable like `Filtered` and maybe also filtering function in a separate variable.

Some of the lines reach 146 characters ... way above 100.